src/Compilers/CSharp/Portable/SymbolDisplay/SymbolDisplayVisitor.Types.cs

```diff
@@ -62,9 +62,22 @@ private void AddArrayRank(IArrayTypeSymbol symbol)
 
             AddPunctuation(SyntaxKind.OpenBracketToken);
 
-            if (insertStars && symbol.Rank > 1)
+            if (symbol.Rank > 1)
             {
-                AddPunctuation(SyntaxKind.AsteriskToken);
+                if (insertStars)
+                {
+                    AddPunctuation(SyntaxKind.AsteriskToken);
+                }
+            }
+            else
+            {
+                var array = symbol as ArrayTypeSymbol;
```

Comment:
In which cases would this be `null`?  Is it when we get the symbol from say a VB compilation?